pyhole/core/plugin.py

```diff
@@ -178,6 +178,7 @@ def load_plugins(*args, **kwargs):
             __import__("pyhole.plugins", globals(), locals(), [plugin_name])
         except Exception, exc:
             LOG.error(exc)
+            raise
```

Comment:
Ah if plugin failure is a warning, then the error should be a warning. We
should probably have a flag for crash on plugin failure.

On Fri, May 20, 2016 at 11:17 AM Josh Kearney <notifications@github.com>
wrote:

> In pyhole/core/plugin.py
> <https://github.com/jk0/pyhole/pull/82#discussion_r64084861>:
>
> > @@ -178,6 +178,7 @@ def load_plugins(*args, **kwargs):
> >              __import__("pyhole.plugins", globals(), locals(), [plugin_name])
> >          except Exception, exc:
> >              LOG.error(exc)
> > +            raise
>
> In this case I'm going to have to object to halting the bot -- the idea is
> that a plugin failure should have no impact on the main bot process.
> Perhaps there is a way to make failures more obvious?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/jk0/pyhole/pull/82/files/d58060b7296214533dab72a7d6d6017152e67ff5#r64084861>
>
-- 
Sent from my mobile 4045076749
Alex Newman